python-squid/src/main/java/org/sonar/python/DocstringExtractor.java

```diff
@@ -36,7 +36,7 @@
  */
 public class DocstringExtractor {
 
-  public static final List<AstNodeType> DOCUMENTABLE_NODE_TYPES = ImmutableList.of(PythonGrammar.FILE_INPUT, PythonGrammar.FUNCDEF, PythonGrammar.CLASSDEF);
+  public static final Set<AstNodeType> DOCUMENTABLE_NODE_TYPES = ImmutableSet.of(PythonGrammar.FILE_INPUT, PythonGrammar.FUNCDEF, PythonGrammar.CLASSDEF);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Convert this Set to an EnumSet. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1641)